contrib/endpoints/src/api_manager/check_security_rules.cc

```diff
@@ -182,39 +128,38 @@ void AuthzChecker::Check(
               // If the parsing of the release body is successful, then call the
               // Test Api for firebase rules service.
               if (status.ok()) {
-                checker->CallTest(ruleset_id, context, final_continuation);
+                checker->request_ = std::unique_ptr<FirebaseRequest>(
+                    new FirebaseRequest(ruleset_id, checker->env_, context));
+                checker->CallNextRequest(final_continuation);
               } else {
                 final_continuation(status);
               }
             });
 }
 
-void AuthzChecker::CallTest(const std::string &ruleset_id,
-                            std::shared_ptr<context::RequestContext> context,
-                            std::function<void(Status status)> continuation) {
-  std::string body;
-  Status status = BuildTestRequestBody(*context.get(), &body);
-  if (!status.ok()) {
-    continuation(status);
+void AuthzChecker::CallNextRequest(
```

Comment:
There is a comment that already exists in the class declaration. look at line 69. Let me know if you need me to add anything more to it.